optimize/self-managed/optimize-deployment/migration-update/3.9-to-3.10-preview.md

```diff
@@ -0,0 +1,23 @@
+---
+id: 3.9-to-3.10-preview
+title: "Update notes (3.9.x to 3.10-preview)"
+---
+
+:::note Heads up!
+To update Optimize to version 3.10-preview, perform the steps in the [migration and update instructions](./instructions.md).
+:::
+
+The update to 3.10.x can be performed from any 3.9.x or any 3.9.0-preview release.
+
+Here you will find information about:
+
+- Changes in the configuration
+
+## Changes in the configuration
+
+Since in the 3.10-preview version of Optimize we have introduced the carbon navigation bar, it is not possible anymore the configure the ui color and color. The following
+environment variables have therefore been deprecated:
```

Comment:
Customers don't need to know that we are using Carbon, that is an implementation detail. In these notes, it's enough to explain what the changes are and their impact, rather than necessarily explain the why and/or how

Reply:
Also they aren't environment variables, they are just Optimize configuration options 🙂